modules/edit-modes/test/translateFromCenter.test.ts

```diff
@@ -0,0 +1,140 @@
+
+import type { Feature as TurfFeature, Geometry } from '@turf/helpers';
+// I used types from geojson because they're generic and helps me with intellisense
+import type {
+  Feature as GenericFeature,
+  Point,
+  MultiPoint,
+  LineString,
+  Polygon,
+  MultiPolygon,
+} from 'geojson';
+import { translateFromCenter } from '../src/translateFromCenter';
+
+type Feature = TurfFeature<Geometry>;
+
+test('Point coordinates in right format', () => {
+  const feature: GenericFeature<Point> = {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [0, 0],
+    },
+    properties: {},
+  };
+  const result = translateFromCenter(
+    feature as Feature,
+    100,
+    100,
+  ) as GenericFeature<Point>;
+  expect(result.geometry.coordinates).toHaveLength(2);
```

Comment:
Typescript already check interfaces, lets check transformation fix instead

Reply:
It's kinda hard to test transformation because this approach doesn't defeat all the distortion - just makes it much smaller and visibly unnoticable, so the geometry wouldn't be strictly the same after moving feature and placing it right back

Reply:
Snapshot testing should work fine